### DISCUSSION TITLE
Versions

### DISCUSSION THREAD

**Discussion by jwmarion** [qiemem/eurorack#56] [General]
Sorry if this is in the wrong place or out of line, but I notice https://github.com/qiemem/eurorack/tree/bipolar/stages#changelog lists up to 1.3.0, but releases stop well short of this. Is there anyway I could get ahold of the audio file (even at my own risk) I'd love to try the flipped layout firmware.

**Comment by qiemem** [comment]
You are in the right place and definitely not out of line! I've been working on the next release and am getting really close. I would actually really appreciate help testing it if you're okay with the risk. Flipped version and regular version attached below. The README should be up to date with all changes in these files. Please let me know if you run into any bugs or issues!

USE AT YOUR OWN RISK, though I will, of course, try to help if anything does go wrong.

[stages-bipolar-flipped-1.3.0-prerelease-3f9fc2b.zip](https://github.com/qiemem/eurorack/files/14964227/stages-bipolar-flipped-1.3.0-prerelease-3f9fc2b.zip)
[stages-bipolar-1.3.0-prerelease-3f9fc2b.zip](https://github.com/qiemem/eurorack/files/14964231/stages-bipolar-1.3.0-prerelease-3f9fc2b.zip)
